### PR TITLE
Why Data Scientists Need to Move from Jupyter Notebooks to Scripts 

### PR DESCRIPTION
**Pull request by EKahuha**
A data scientist interested in using scientific methods, algorithms and processes to extract insights from both structural and unstructured data. Enjoys converting raw data into meaningful information and contributing to data science topical issues.

### CODE REVIEW

**Comment by hectorkambow** [comment]
Just started a preliminary review on your article - this being your 1st article if you could please include a headshot image and short bio to include with your articles ( they will appear at the bottom of all your articles)

Please see any published article for an example. 

You can submit the .md file and image as a new PR - or you can include them both on this PR via a comment - and I will add them. 
Thank you. 
@EKahuha

**Comment by EKahuha** [comment]
**Bio**
Eric is a data scientist interested in using scientific methods, algorithms and processes to extract insights from both structural and unstructured data. Enjoys converting raw data into meaningful information and contributing to data science topical issues. 
  
Eric Kahuha- Headshot Image

**Comment by hectorkambow** [comment]
Thank you for the bio and headshot image  - 
just wanted to double-check with you on the in-article images - if they were screenshots from your computer - or if they needed an image source. 
🤔

**Comment by EKahuha** [comment]
> Thank you for the bio and headshot image -
> just wanted to double-check with you on the in-article images - if they were screenshots from your computer - or if they needed an image source.
>  @EKahuha

They are screenshots from my computer. Thanks.  
@hectorkambow

File: articles/why-data-scientists-need-to-move-from-jupyter-notebooks-to-scripts/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+A data engineer may want to test different data processing methods by choosing several parameters for his or her machine-learning algorithm to test an increase accuracy. However, every time they experiment with new approaches, they have to find and rerun the associated cells. In the end, these scientists find it time-consuming and confusing, especially when the training or processing procedure takes a long run time.
+
+Moreover, Jupyter Notebook does not support reproducibility. It becomes increasingly difficult to use new data with slightly different structures with Jupyter Notebook. This is because of the challenges involved in identifying the source of the error in the notebook. Not only that, but Jupyter Notebook can make it challenging to debug. In case of an error in a written code, it is difficult to determine whether the reason for the error is the change in data or the code. With Jupyter Notebook not integrating very well with other tools, production is difficult. For example, you cannot run Jupyter Notebook code easily while using other codes.
```
Comment (hectorkambow) [inline articles/why-data-scientists-need-to-move-from-jupyter-notebooks-to-scripts/index.md:27]: For this last sentence: "For example, you cannot run Jupyter Notebook code easily while using other codes."

I can not help but think the audience may be left wondering of a use case example - if you could include (ideal a short snippet) but depending on what you meant -even a short description of an issue or use case - would help the reader imagine what or type of scenario/problem you have in mind.

**Comment by hectorkambow** [comment]
Also upon closer review of the article - if you could please revisit the article - and change much of the content:
It is too close in resemblance to this article you referenced:
https://towardsdatascience.com/5-reasons-why-you-should-switch-from-jupyter-notebook-to-scripts-cb3535ba9c95
Your article needs to be sufficiently different than any other content - 
Only original content please 🚀 @EKahuha 
The structure and content of your article is too similar - to the press Shift + Enter to the reasons. 
Id suggest finding better reasons why you personally went from Jupyter to scripts - and perhaps a personal project example that led you to that decision - 👍

**Comment by hectorkambow** [comment]
Closing to make room in the PR queue - CAN BE REOPENED AT ANY TIME -

**Pull request closed without merging**